Laud_Misc_610/CGH/_1.trig

```diff
@@ -928,28 +951,105 @@
 <#Trian>
     a foaf:Person;
     irishRel:nomName "Trian";
-    rel:childOf <#Níall>;
+    rel:childOf <#Níall-444391bc>;
     owl:sameAs <http://example.com/Rawl_B502/mínigud_senchais_síl_chuind_inso_sís.trig#Trian>.
 
 <#Húathgen>
     a foaf:Person;
     irishRel:nomName "Húathgen";
-    rel:childOf <#Níall>;
+    rel:childOf <#Níall-444391bc>;
     owl:sameAs <http://example.com/Rawl_B502/mínigud_senchais_síl_chuind_inso_sís.trig#Uaithgein>.
 
 <#Coíldub>
     a foaf:Person;
     irishRel:nomName "Coíldub";
-    rel:childOf <#Níall>;
+    rel:childOf <#Níall-444391bc>;
     owl:sameAs <http://example.com/Rawl_B502/mínigud_senchais_síl_chuind_inso_sís.trig#Cóeldub>.
 
+<#ConalleBregh>
```

Comment:
The name in the text is "Conall Bregh" so perhaps change this to `<#ConallBregh>`, here and below?